cds_books/literature/ext.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019-2020 CERN.
+#
+# CDS Books is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""Literature module."""
+
+
+from invenio_records.signals import after_record_insert, after_record_update, \
+    before_record_insert, before_record_update
+
+from .tasks import set_cover, update_cover
+
+
+class CdsBooksLiterature(object):
```

Comment:
I would go for maybe creating a CdsIls extension instead, a main one.

Reply:
Yes that makes sense, I went for `CdsBooks`, to match every other occurrence. We can move the repository and rename everything in one go.